### PR TITLE
Task merge unpack8bit

### PR DESCRIPTION
**Pull request by spawlows**

### CODE REVIEW

**Comment by hassount** [comment]
lgtm,
could you please address the linux warning?

**Comment by spawlows** [comment]
linux warning fixed

File: Source/Lib/ASM_AVX2/EbPackUnPack_Intrinsic_AVX2.c [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+    uint32_t x, y;
+
+    __m256i ymm_00FF, in_pixel0, in_pixel1;
+    __m256i in_pixel1_shftR_2_U8, in_pixel0_shftR_2_U8;
+    ymm_00FF = _mm256_set1_epi16(0x00FF);
+
+    if (width == 8)
+    {
+        __m128i xmm_00FF, in_pixel0, in_pixel1, in_pixel1_shftR_2_U8;
+        __m128i in_pixel0_shftR_2_U8, in_pixel0_shftR_2, in_pixel1_shftR_2;
```
Comment (joelsole) [inline Source/Lib/ASM_AVX2/EbPackUnPack_Intrinsic_AVX2.c]: braces according to style.md:

if (condition) {
    do_something_1();
    do_something_2();
}

File: Source/Lib/ASM_AVX2/EbPackUnPack_Intrinsic_AVX2.c [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+        xmm_00FF = _mm_set1_epi16(0x00FF);
+        for (y = 0; y < height; y += 2)
+        {
+            in_pixel0 = _mm_loadu_si128((__m128i*) in_16bit_buffer);
```
Comment (joelsole) [inline Source/Lib/ASM_AVX2/EbPackUnPack_Intrinsic_AVX2.c]: braces:

if (condition) {
    do_something_1();
    do_something_2();
}

ditto below

File: Source/Lib/Codec/EbPackUnPack.h [hunk 3]
```diff
@@ -11,6 +11,7 @@ extern "C" {
 
 #include "EbPackUnPack_C.h"
 #include "EbPackUnPack_SSE2.h"
```
Comment (joelsole) [inline Source/Lib/Codec/EbPackUnPack.h]: is the include "EbPackUnPack_SSE2.h" still needed?

File: Source/Lib/ASM_AVX2/EbPackUnPack_Intrinsic_AVX2.c (hunk 2 above)
Comment (spawlows) [reply Source/Lib/ASM_AVX2/EbPackUnPack_Intrinsic_AVX2.c]: done

File: Source/Lib/Codec/EbPackUnPack.h (hunk 3 above)
Comment (spawlows) [reply Source/Lib/Codec/EbPackUnPack.h]: Yes, There are other used functions in "EbPackUnPack_SSE2.h".

**Pull request merged**